src/primitive.js

```diff
@@ -768,6 +768,12 @@ var _ = Mavo.Primitive = class Primitive extends Mavo.Node {
 				let presentational;
 
 				if (this.options) {
+					if (!this.options.has(value)) {
+						// There is no option corresponding to the value. Try the default one.
+						// If there is no option corresponding to the default value either, use the first option
+						value = this.options.has(this.default)? this.default : this.options.keys().next().value;
```

Comment:
We should not change the value if it doesn't match any of the options, as that's destructive. The user may be experimenting with a different UI, or have forgotten to add all pertinent options. As a design principle, loading data into *any* Mavo and then saving it back should be a non-destructive operation. 

For `<select>` there is a whole code path that creates (and removes when no longer needed) `.mv-volatile` options in the `<select>` to deal with this. `mv-options` should behave similarly.

Reply:
Possibly not for v0.3.0 then?

Reply:
> Possibly not for v0.3.0 then?

I was going to suggest this. I might have been a bit too optimistic about the issue. 😅